nautobot/extras/templates/extras/plugin_detail.html

```diff
@@ -0,0 +1,247 @@
+{% extends 'generic/object_detail.html' %}
+{% load helpers %}
+
+{% block header %}
+    <div class="row noprint">
+        <div class="col-md-12">
+            <ol class="breadcrumb">
+                {% block breadcrumbs %}
+                    <li><a href="{% url 'plugins:plugins_list' %}">Installed Plugins</a></li>
+                    <li><a href="{% url 'plugins:plugin_detail' plugin=object.name %}">{{ object.verbose_name | bettertitle }}</a></li>
+                {% endblock breadcrumbs %}
+            </ol>
+        </div>
+    </div>
+    <div class="pull-right noprint">
+        {% block buttons %}
+            {% if object.home_view_name %}
+                <a href="{% url object.home_view_name %}" class="btn btn-success">
+                    <i class="mdi mdi-home"></i> Plugin Home
+                </a>
+            {% endif %}
+            {% if object.config_view_name %}
+                <a href="{% url object.config_view_name %}" class="btn btn-warning">
+                    <i class="mdi mdi-cog"></i> Plugin Configuration
+                </a>
+            {% endif %}
+        {% endblock buttons %}
+    </div>
+    {% block masthead %}
```

Comment:
Just for me: where does this block come from?

Reply:
It comes from the parent template, nautobot/core/templates/generic/object_detail.html. I could also have just overridden the inner `block title`, I suppose, as there's no change to the outer `<h1>...</h1>` that `block masthead` provides by default.